docs/kaust.md

```diff
@@ -24,20 +24,35 @@ module load nextflow
 Launch the pipeline with `-profile kaust` (one hyphen) to run the workflows using the KAUST profile.
 This will download and launch the [`kaust.config`](../conf/kaust.config) which has been pre-configured with a setup suitable for the KAUST servers.
 It will enable `Nextflow` to manage the pipeline jobs via the `Slurm` job scheduler and `Singularity` to run the tasks.
-Using the KAUST profile, `Docker` image(s) containing required software(s) will be downloaded, and converted to `Singularity` image(s) if needed before execution of the pipeline. To avoid downloading same images by multiple users, we provide a singularity `libraryDir` that is configured to use images already downloaded in our central container library. Images missing from our library will be downloaded to your home directory path as defined by `cacheDir`.
+Using the KAUST profile, `Docker` image(s) containing required software(s) will be downloaded, and converted to `Singularity` image(s) if needed before execution of the pipeline. To avoid downloading same images by multiple users, we provide a singularity `libraryDir` that is configured to use images already downloaded in our central container library. Images missing from our library will be downloaded to the user's directory as defined by `cacheDir`.
+
+### Accessing reference genomes on Ibex
+
+We provide a collection of reference genomes, enabling users to run workflows seamlessly without needing to download the files. To enable access to this resource, simply add the `includeConfig` line in the script below to a `nextflow.config` file under the launch directory.
```

Comment:
Are you a system administration?

It might be easier to add to all users home directory a `.nextflow/config` file which contains this (rather than users having to make their own each time), otherwise we can discuss other options.

The `.nbextflow./config` file gets implicityly loaded by all nextflow runs, which is why it's a 'nice' option

Reply:
Nope and I don't think they would accept that since the genomes.config file will be changed by us frequently!